corehq/apps/reports/filters/users.py

```diff
@@ -406,28 +406,28 @@ def user_es_query(cls, domain, mobile_user_and_group_slugs, request_user):
             location_query = location_query.accessible_to_user(domain, request_user)
         location_ids = list(location_query.location_ids())
 
-        group_id_filter = filters.term("__group_ids", group_ids)
-
         if toggles.FILTER_ON_GROUPS_AND_LOCATIONS.enabled(domain) and group_ids and location_ids:
             group_and_location_filter = filters.AND(
-                group_id_filter,
+                filters.term("__group_ids", group_ids),
                 user_es.location(location_ids),
```

Comment:
shouldn't we also be doing the user is active filtering here to keep behavior consistent?

Reply:
Yep!  And to that end I actually moved the `is_active` check outside of the `if/else`, to apply to the overall `group_and_location_filter` (guessing this comment was from an earlier commit).  It should now apply to both groups and locations with or without the FF enabled.